src/commands/TrackerCommands.ts

```diff
@@ -0,0 +1,154 @@
+import { Command } from "../interactions/interactionClasses";
+import {
+	ChatInputCommandInteraction,
+	EmbedBuilder,
+	SlashCommandBuilder,
+	SlashCommandSubcommandsOnlyBuilder,
+} from "discord.js";
+import { latest, logs, playtime, stats } from "../util/tracker/subCommands";
+import { userLast, userStats, userTop } from "../util/tracker/userCommands";
+import { gameLast, gameStats, gameTop } from "../util/tracker/gameCommands";
+import { addBlacklist, remBlacklist } from "../util/tracker/blacklistCommands";
+import { config } from "../config";
+
+class Tracker extends Command {
+	constructor() {
+		super("tracker");
+	}
+
+	async execute(interaction: ChatInputCommandInteraction): Promise<void> {
+		if (!config.tracking) {
+			await interaction.reply({
+				embeds: [
+					new EmbedBuilder()
+						.setTitle("tracking disabled")
+						.setDescription(
+							"Tracking is disabled. No game activity's will be logged and tracking commands are disabled."
+						)
+						.setColor([255, 0, 0]),
```

Comment:
For a quick message like this embeds are overkill IMO. Please just make this a simple message. Embeds should belong to bigger things